src/main/scala/nl/biopet/utils/Histogram.scala

```diff
@@ -118,8 +122,9 @@ object Histogram {
     (for ((name, idx) <- header)
       yield
         name -> {
-          new Histogram[T](
-            values.flatMap(x => x._2.lift(idx).flatten.map(x._1 -> _)))
+          new Histogram[T](values.flatMap {
+            case (k, v) => v.lift(idx).flatten.map(k -> _)
```

Comment:
Can you give a more descriptive name for k and v?
Given a histogram, something like `label` and `value` might be more descriptive.